src/app/components/Slider/Slider.tsx

```diff
@@ -38,7 +38,7 @@ const Rangeslider = ({
         className={styles.slider}
         value={value}
         id={id}
-        onChange={(event) => onChange(event.target.value)}
+        onChange={onChange}
```

Comment:
I liked the old solution more, I'm not a fan of passing the event itself

Reply:
Sorry, but I cant resolve this problem